README.md

```diff
@@ -43,6 +43,36 @@ or from PyPI:
 pip install optax
 ```
 
+## Quick Start
+
+Optax contains implementations of [many popular optimizers](https://optax.readthedocs.io/en/latest/api.html#Common-Optimizers). For example
+the following code snippet obtains `adam`. We initialize the optimizer state
+using the `init` function and `params` of the network.
+
+```
+optimizer = optax.adam(learning_rate)
+# Obtain the `opt_state` that contains statistics for the optimizer.
+opt_state = optimizer.init(params)  # params are the weights you're optimizing.
+```
+
+To write the update loop we need a loss function (here assumed to be
+`compute_loss`) that can differentiated by `jax` to obtain the gradients.
```

Comment:
Suggestion: add missing "be" and emphasize `jax.grad` since the other alternative could be `jax.value_and_grad` 🤷‍♂️ 

```suggestion
`compute_loss`) that can be differentiated by JAX (with `jax.grad` in this example) to obtain the gradients.
```